packages/cli/src/config/config.ts

```diff
@@ -155,6 +157,11 @@ async function parseArguments(): Promise<CliArgs> {
       type: 'string',
       description: 'Allowed MCP server names',
     })
+    .option('max-turns', {
```

Comment:
Should this be a command line flag or a settings.json piece? I think I'd opt for the latter because it only impacts command line interactions that are `-p` or full run based. For instance, interacting with --version or extension commands don't make sense.